indico/queries/forms.py

```diff
@@ -58,3 +56,58 @@ def requests(self):
         else:
             yield UploadDocument(files=self.files)
         yield _FormPreprocessing(files=self.previous)
+
+
+class ListPrebuiltForms(GraphQLRequest):
+
+    query = """
+    mutation listPrebuiltFormsMutation {
+        listPrebuiltForms {
+            forms {
+                id
+                name
+            }
+        }
+    }
+    """
+    method = HTTPMethod.GET
+
+    def __init__(self):
+        super().__init__(query=self.query)
+
+    def process_response(self, response):
+        return super().process_response(response)["listPrebuiltForms"]["forms"]
+
+
+class GetPrebuiltForm(GraphQLRequest):
+
+    query = """
+    mutation($form_id: Int!) {
```

Comment:
Same as above I think there some confusion on the way graphql works if these are built as mutations.